app/src/main/res/values-ar/strings.xml

```diff
@@ -498,19 +498,31 @@
+  <string name="faq_answer_math_lessons" fuzzy="true"/>
+  <string name="faq_answer_more_lessons" fuzzy="true"/>
   <string name="faq_answer_exploration_player">&lt;p&gt;إذا لم يتم تحميل مشغل الاستكشاف&lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;تحقق لمعرفة ما إذا كان التطبيق محدثًا أم لا:&lt;/p&gt;&lt;p&gt; &lt;ul&gt; &lt;li&gt; انتقل إلى متجر Play وتأكد من تحديث التطبيق إلى أحدث إصدار &lt;/li&gt;&lt;/ul&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;تحقق من اتصالك بالإنترنت:&lt;/p&gt;&lt;ul&gt;&lt;li&gt; إذا كان اتصالك بالإنترنت بطيئًا ، فحاول إعادة الاتصال بشبكة Wi-Fi أو الاتصال بشبكة أخرى. &lt;/li&gt;&lt;/ul&gt;&lt;p&gt;اطلب من المشرف التحقق من أجهزتهم واتصال الإنترنت:&lt;/p&gt;&lt;ul&gt;&lt;li&gt; اطلب من المشرف استكشاف الأخطاء وإصلاحها باستخدام الخطوات المذكورة أعلاه &lt;/li&gt;&lt;/ul&gt;&lt;p&gt;أخبرنا إذا كنت لا تزال تواجه مشكلات في التحميل:&lt;/p&gt;&lt;ul&gt;&lt;li&gt; أبلغ عن مشكلة عن طريق الاتصال بنا على admin@oppia.org. &lt;/li&gt;&lt;/ul&gt;</string>
   <string name="faq_answer_audio_not_playing">&lt;p&gt;إذا لم يتم تشغيل الصوت الخاص بك&lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;تحقق لمعرفة ما إذا كان التطبيق محدثًا أم لا:&lt;/p&gt;&lt;ul&gt;&lt;li&gt;انتقل إلى متجر Play وتأكد من تحديث التطبيق إلى أحدث إصدار&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;تحقق من اتصالك بالإنترنت:&lt;/p&gt;&lt;ul&gt;&lt;li&gt;إذا كان اتصالك بالإنترنت بطيئًا ، فحاول إعادة الاتصال بشبكة Wi-Fi أو الاتصال بشبكة أخرى. قد يتسبب الإنترنت البطيء في تحميل الصوت بشكل غير منتظم ، مما يجعل من الصعب تشغيله.&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;اطلب من المسؤول التحقق من أجهزتهم واتصال الإنترنت:&lt;/p&gt;&lt;ul&gt;&lt;li&gt;اطلب من المسؤول استكشاف الأخطاء وإصلاحها باستخدام الخطوات المذكورة أعلاه&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;أخبرنا إذا كنت لا تزال تواجه مشكلات في التحميل:&lt;/p&gt;&lt;ul&gt;&lt;li&gt;أبلغ عن مشكلة عن طريق الاتصال بنا على admin@oppia.org.&lt;/li&gt;&lt;/ul&gt;</string>
+  <string name="faq_answer_delete_profile">&lt;p&gt;بمجرد حذف ملف التعريف:&lt;/p&gt;&lt;ol&gt;&lt;li&gt;لا يمكن استعادة ملف التعريف.&lt;/li&gt;&lt;li&gt;سيتم حذف معلومات الملف الشخصي مثل الاسم والصور والتقدم بشكل دائم.&lt;/li&gt;&lt;/ol&gt;&lt;p&gt;لحذف ملف تعريف (باستثناء&lt;u&gt;المسؤول&lt;/u&gt;):&lt;/p&gt;&lt;ol&gt;&lt;li&gt; من الصفحة الرئيسية للمسؤول ، اضغط على زر القائمة أعلى اليسار.&lt;/li&gt;&lt;li&gt; اضغط على&lt;strong&gt;ضوابط المسؤول&lt;/strong&gt;. &lt;/li&gt;&lt;li&gt; اضغط على&lt;strong&gt;تحرير ملفات التعريف&lt;/strong&gt;.&lt;/li&gt;&lt;li&gt; اضغط على الملف الشخصي الذي ترغب في حذفه.&lt;/li&gt;&lt;li&gt; في الجزء السفلي من الشاشة ، انقر فوق&lt;strong&gt;حذف الملف الشخصي&lt;/strong&gt;. &lt;/li&gt;&lt;li&gt; اضغط&lt;strong&gt;حذف&lt;/strong&gt;لتأكيد الحذف. &lt;/li&gt;&lt;/ol&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt; ملاحظة:&lt;u&gt;المسؤول&lt;/u&gt;فقط هو القادر على إدارة الملفات الشخصية.&lt;/p&gt;</string>
+  <string name="faq_answer_update_app" fuzzy="true"/>
```

Comment:
These two look like empty strings.

Reply:
I looked up this one and it they got emptied on translatewiki.net when https://github.com/oppia/oppia-android/commit/30ba44dc91e42d0fd960aac9306f76170d3ba3e4#diff-275e22e5424874865d61be0a790c48168da5998ec6dd530c83fb33005bfa4449R524 was imported. My guess is that the unescaped html in the messages choke our parser which caused the strings to be empty. This is a bug on our side: we are using PHP's `SimpleXMLElement` which doesn't provide method to get innerHTML.

This is probably the cause for all other empty strings too

Reply:
> I looked up this one and it they got emptied on translatewiki.net when [30ba44d#diff-275e22e5424874865d61be0a790c48168da5998ec6dd530c83fb33005bfa4449R524](https://github.com/oppia/oppia-android/commit/30ba44dc91e42d0fd960aac9306f76170d3ba3e4#diff-275e22e5424874865d61be0a790c48168da5998ec6dd530c83fb33005bfa4449R524) was imported. My guess is that the unescaped html in the messages choke our parser which caused the strings to be empty. This is a bug on our side: we are using PHP's `SimpleXMLElement` which doesn't provide method to get innerHTML.
> 
> This is probably the cause for all other empty strings too

Thanks @Nikerabbit. Do you have a suggestion for how we can work around this at the moment?

Reply:
The values should re restored with escaping or directly in translatewiki.net

Reply:
@Nikerabbit thanks for the earlier insight. We've now submitted a change that should be correctly escaping HTML tag elements rather than using CDATA (see #5524), but it seems this issue is still happening. Does this mean we'll need to fix the strings on the Translatewiki side or is there still something problematic with our string formatting?

Reply:
The old translations are available via the history of individual pages listed in
* https://translatewiki.net/w/i.php?title=Special:Translations&message=Oppia%3Aoppia-android-app-faq_answer_app_language
* https://translatewiki.net/w/i.php?title=Special:Translations&message=Oppia%3Aoppia-android-app-faq_answer_update_app

It seems that both `faq_answer_app_language` and `faq_answer_bug_reporting` have changed in their content, so maybe it would be better if I just deleted the broken translations?

Reply:
It might be okay to do that.